vignettes/lab_grading.Rmd

```diff
@@ -0,0 +1,395 @@
+---
+title: "Lab grading"
```

Comment:
![image](https://user-images.githubusercontent.com/10111024/187245945-23b450c0-10a0-409e-a1a1-52e216015286.png)

We should say Lab Grading

Reply:
Done